src/test/java/com/dao/DishOrderDAOTest.java

```diff
@@ -0,0 +1,97 @@
+package com.dao;
+
+import com.model.DishOrder;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.*;
+
+import javax.sql.DataSource;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DishOrderDAOTest {
+
+    DishOrderDAO dishOrderDAO = DishOrderDAO.getInstance();
+
+    @Before
+    public void init() throws Exception {
+
+        dishOrderDAO.setDataSource(getDataSource());
+    }
+
+    public DataSource getDataSource() {
+
+        EmbeddedDatabaseBuilder builder = new EmbeddedDatabaseBuilder();
+
+        return builder
+                .setType(EmbeddedDatabaseType.H2)
+                .addScript("init_dish_order.sql")
+                .addScript("data_dish_order.sql")
+                .build();
+    }
+
+    @Test
+    public void add() throws Exception {
+
+        DishOrder dishOrder = new DishOrder(6, 100, 100, 100, 1000);
+        dishOrderDAO.create(dishOrder);
+
+        assertEquals(dishOrderDAO.getByOrderId(100).get(), dishOrder);
+    }
+
+    @Test
+    public void update() throws Exception {
+
+        DishOrder act = dishOrderDAO.getById(1).get();
+        act.setDishSum(100);
+        dishOrderDAO.update(act);
+
+        DishOrder exp = dishOrderDAO.getById(1).get();
+        assertEquals(act, exp);
+    }
+
+    @Test
+    public void remove() throws Exception {
+
+        assertEquals(true, dishOrderDAO.getById(2).isPresent());
```

Comment:
use assertTrue and assertFalse
and more desirable, use hamcrest assert matchers